check_raid.pl

```diff
@@ -559,15 +558,41 @@ sub program_names {
 
 sub commands {
 	{
-		'status' => ['-|', '@CMD'],
+		'status' => [>&2', '@CMD'],
```

Comment:
this looks syntax error. what's your intention? does the metastat tools print data to stderr?